morpheushelper/cogs/invites.py

```diff
@@ -178,7 +176,7 @@ async def invites_show(self, ctx: Context, *, invite: AllowedServerConverter):
         invite_guild = await self.check_invite(invite.code)
         if invite_guild is not None:
             invite_title = translations.invite_link
-            embed.set_thumbnail(url=invite_guild.guild.icon_url_as(size=128))
+            embed.set_thumbnail(url=invite_guild.guild.icon_url)
```

Comment:
there was actually a reason why I wanted the icon url to be small, as there are sever, whose icon is too big for the file size and therefore not loaded. To reproduce: https://discord.gg/D4H5rATmg2